launcher/ui/pages/global/APIPage.ui

```diff
@@ -147,6 +147,58 @@
          </layout>
         </widget>
        </item>
+      <item>
+        <widget class="QGroupBox" name="groupBox_cfproxy">
+         <property name="title">
+          <string>&amp;CurseForge API</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_41">
+          <item>
+           <widget class="Line" name="line_21">
+            <property name="orientation">
+             <enum>Qt::Horizontal</enum>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QLabel" name="label_31">
+            <property name="text">
+             <string>Note: you probably don't need to set this if Curseforge mod downloading already works. Using the Curseforge API directly is unsupported, because we currently don't have an option to set an API key.</string>
```

Comment:
Maybe like this instead

```suggestion
             <string>Note: you probably don't need to set this if Curseforge mod downloading already works. Using the Curseforge API directly is unsupported, as it would require an API key to work.</string>
```